units/scheduler_alias.go

```diff
@@ -0,0 +1,103 @@
+package units
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/distro"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/scheduler"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+const schedulerAliasJobName = "distro-alias-scheduler"
+
+func init() {
+	registry.AddJobType(schedulerAliasJobName, func() amboy.Job {
+		return makeDistroAliasSchedulerJob()
+	})
+}
+
+type distroAliasSchedulerJob struct {
+	DistroID string `bson:"distro_id" json:"distro_id" yaml:"distro_id"`
+	job.Base `bson:"metadata" json:"metadata" yaml:"metadata"`
```

Comment:
We use `job_base` everywhere else.

Reply:
we are unfortunately split on this question about 50/50... I don't think it matters at all.